browser/ui/views/infobars/BUILD.gn

```diff
@@ -3,55 +3,63 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
```

Comment:
No error w/o `import("//brave/components/brave_wayback_machine/buildflags/buildflags.gni")` in this file?

Reply:
yes, and fixed